release/src/Upload.hs

```diff
@@ -0,0 +1,71 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE MultiWayIf #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE StrictData #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+module Upload (
+    uploadToMavenCentral
+) where
+
+import qualified Control.Exception.Safe as E
+import           Control.Monad.Logger
+import           Control.Monad.IO.Class
+import           Data.Foldable (for_)
+import           Data.List (foldl)
+import           Network.HTTP.Client (newManager, httpNoBody)
+import           Network.HTTP.Client.TLS (tlsManagerSettings)
+import           Network.HTTP.Simple (getResponseStatusCode, parseRequest, setRequestBasicAuth, setRequestBodyFile, setRequestHeader, setRequestMethod, setRequestPath)
+import           Data.Text (Text, pack)
+import           Data.Text.Encoding (encodeUtf8)
+import           Path
+import           Path.IO
+
+import Types
+import Util (BazelLocations(..))
+
+uploadToMavenCentral ::(MonadLogger m, MonadIO m, E.MonadThrow m) => PerformUpload -> BazelLocations -> [(MavenCoords, Path Rel File)] -> m ()
+uploadToMavenCentral upload BazelLocations{..} artifacts = do
+
+    manager <- liftIO $ newManager tlsManagerSettings
+
+    initialRequest' <- parseRequest "http://localhost:8081"
```

Comment:
Looks like that URL is probably left over from testing?

Reply:
Correct. I haven't added the configuration handling as yet.